_targets_eval_postprocessing.R

```diff
@@ -388,6 +388,45 @@ head_to_head_targets <- list(
   )
 )
 
+# Manuscript figures------------------------------------------------
+# Note that these are just the components of the figures, not the full
+# ggarranged, properly formatted figures, and currently require
+# specification for the figure components that are examples.
+manuscript_figures <- list(
+  tar_target(
+    name = fig2_hosp_t,
+    command = make_fig2_hosp_t(
+      hosp_quantiles_filtered,
+      locs_to_plot = c("MA", "AL", "WA"),
+      date_to_plot = "2024-01-15"
+    )
+  ),
+  tar_target(
+    name = fig2_ct_1,
+    command = make_fig2_ct(
+      all_ww_quantiles_sq,
+      loc_to_plot = "MA",
+      date_to_plot = "2024-01-15"
+    )
+  ),
+  tar_target(
+    name = fig2_ct_2,
+    command = make_fig2_ct(
+      all_ww_quantiles_sq,
+      loc_to_plot = "AL",
+      date_to_plot = "2024-01-15"
+    )
+  ),
+  tar_target(
+    name = fig2_ct_3,
+    command = make_fig2_ct(
+      all_ww_quantiles_sq,
+      loc_to_plot = "WA",
+      date_to_plot = "2024-01-15"
+    )
```

Comment:
Having the concentration plots be one ggplot per location while the forecast plots are a single faceted ggplot might make axis alignment, etc down the road more difficult. 

Related: as written, this produces different plot aspects depending on the number of wastewater sites within a location. If you're going to do each location individually, you'll probably want functionality to add null "site" facets for locations with fewer than the maximum number of plotted sites, so that the x axes can be aligned.

Reply:
True, my thinking was whenever we can to use faceting, I just didn't do it for the wastewater sites because of the different number of sites thing. 

I think of this as a bit of a downstream formatting question e.g. do we want to show a set number of sites even if the state has a different number of sites, or do we want to always show all sites? 

Defer to you here on how we want to do this in the short term -- can either make them all separate plots or set up faceting now adding null "site" facets and or enforcing a set number of sites to be displayed